src/cabal

```diff
@@ -118,6 +120,19 @@ run_quick () {
 }
 
 #
+# Load up ghcid
+#
+run_watch () {
+    initialize
+    [ $# -eq 1 ] || fail "Watch requires a single argument, specifying the entry point you want to load."
+    [ -f "$1" ] || fail "The entry point does not exist."
+    [ ! -d src ] || SRC_DIRS="-isrc"
+    [ ! -d test ] || SRC_DIRS="${SRC_DIRS:-} -itest"
+    [ ! -d dist/build/autogen ] || SRC_DIRS="${SRC_DIRS:-} -idist/build/autogen"
+    ghcid -c "ghci -package-db=$(find .cabal-sandbox -name '*-packages.conf.d') ${SRC_DIRS:-} \"$1\""
```

Comment:
Could also make a function that generates the `ghci` command string and re-use in quick and here if you feel like it. :)